app/models/cohort_columns/client_id.rb

```diff
@@ -0,0 +1,25 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module CohortColumns
+  class ClientId < ReadOnly
+    attribute :column, String, lazy: true, default: :client_id
+    attribute :translation_key, String, lazy: true, default: 'Client ID'
+    attribute :title, String, lazy: true, default: ->(model, _attr) { _(model.translation_key) }
+
+    def value(cohort_client)
+      cohort_client.client.id
```

Comment:
This will force a join to GrdaWarehouse::Hud::Client.  You should be able to get the id from `cohort_client.client_id` which is stored in the cohort_clients table.

Reply:
Makes sense. I'll fix it up.